src/Services/INomadRepoService.cs

```diff
@@ -0,0 +1,17 @@
+﻿using WindowsAppCommunity.Sdk.Nomad;
+
+namespace WindowsAppCommunity.Discord.ServerCompanion.Services
+{
+    public interface INomadRepoService
+    {
+        public Task<(RepositoryContainer, WacsdkCommandConfig, WacsdkNomadSettings)> GetNomadRepoAsync(string repoId, string knownId, CancellationToken token);
```

Comment:
We'll need to remove `WacsdkCommandConfig` from this interface. It's a high-level object that encodes runtime options, not specific to any one repo or operator.

Reply:
We also don't need the `knownId` to get the repo itself. Known IDs are used *within* a repo to locate registered data, it's not used to get the repo itself.